addons/website_project/controllers/main.py

```diff
@@ -31,6 +31,9 @@ def extract_data(self, model, values):
         if model.model == 'project.task' and values.get('email_from'):
             partners_list = request.env['mail.thread'].sudo()._mail_find_partner_from_emails([values['email_from']])
             partner_id = partners_list[0].id if partners_list else False
+            data['record']['partner_name'] = values['partner_name']
+            data['record']['partner_phone'] = values['partner_phone']
+            data['record']['partner_company_name'] = values['partner_company_name']
```

Comment:
If the partner exists, then we should just set those data only if the field is empty on the partner record.

What do you think? 

Otherwise, we will each time erase the data once the customer submit a task/ticket.

Reply:
> If the partner exists, then we should just set those data only if the field is empty on the partner record.
> 
> What do you think?
> 
> Otherwise, we will each time erase the data once the customer submit a task/ticket.

If the customer or user adds new data, such as an email, company name , we should store the new data in the task. Why should we use old data?

Reply:
Because it is untrusted data and so the customer or any user could easily change the partner data, each time he submits a ticket/task.